packages/scripts/scripts/check-engines.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+const { sync: spawn } = require( 'cross-spawn' );
+const { sync: resolveBin } = require( 'resolve-bin' );
+
+/**
+ * Internal dependencies
+ */
+const {
+	getCliArgs,
+	hasCliArg,
+} = require( '../utils' );
+
+const args = getCliArgs();
+
+const hasConfig = hasCliArg( '--package' ) ||
+	hasCliArg( '--node' ) ||
+	hasCliArg( '--npm' ) ||
+	hasCliArg( '--yarn' );
+const config = ! hasConfig ?
+	[
+		'--node', '>=8.0.0',
```

Comment:
```suggestion
		'--node', '>=10.0.0',
```

As _core_ has switched to using Node.js 10.x we should also use 10.x here

Reply:
I wasn't sure. Sounds awesome 👍
Was it announced somewhere? It would be nice to just merge it :)